test_scripts/API/Progress_Bar_Seek_Feature/commonProgressBarSeek.lua

```diff
@@ -0,0 +1,106 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+
+--[[ Module ]]
+local c = actions
+
+--[[ Variables ]]
+
+c.requestParams = {
+  startTime = {
+    hours = 0,
+    minutes = 1,
+    seconds = 33
+  },
+  endTime = {
+    hours = 0,
+    minutes = 59 ,
+    seconds = 35
+  },
+  updateMode = "COUNTUP"
+}
+
+c.seekTimeParams = {
+	hours = 0,
+	minutes = 1,
+	seconds = 1
+}
+
+--[[ Common Functions ]]
+
+--[[ @SetMediaClockTimer: Successful processing SetMediaClockTimer RPC
+--! @parameters:
+--! pValue - value for enableSeek parameter
+--! @return: none
+--]]
+function c.SetMediaClockTimer(pValue)
+  c.requestParams.enableSeek = pValue
+  local cid = c.getMobileSession():SendRPC("SetMediaClockTimer", c.requestParams)
+
+  c.requestParams.appID = c.getHMIAppId()
+  c.getHMIConnection():ExpectRequest("UI.SetMediaClockTimer", c.requestParams)
+  :Do(function(_, data)
+    c.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+  end)
+  c.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+end
+
+--[[ @setMediaClockTimerUnsuccess: Processing SetMediaClockTimer with ERROR resultCode
+--! @parameters:
+--! pValue - value for enableSeek parameter
+--! pResultCode - result error
+--! @return: none
+--]]
+function c.setMediaClockTimerUnsuccess(pValue, pResultCode)
```

Comment:
@HSavynetska Please use the same name pattern for functions - 1st letter in upper case or lowercase. Now there is a mix: (`SetMediaClockTimer` vs `setMediaClockTimerUnsuccess`)